### PR TITLE
Extend testing README.md for external contributors without admin rights

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ They are run on **every** PR by default.
 - **none**
 
 ### FAQ
+- *I am not part of the `coreos` or `coreos-inc` GitHub organization. Why are
+  the tests not being executed on my changes?*
+
+  Pull requests by external contributors need to be approved before they are
+  tested by our Jenkins setup. Please ask a maintainer to approve your PR via
```
Comment (cpanato) [inline tests/README.md]: This step the maintainer is not approving the PR yet, he/she will just enable the tests to be executed. I guess after that the maintainer will approve the PR.

Comment (mxinden) [reply tests/README.md]: That's an important fact to clarify.

**Comment by mxinden** [comment]
@cpanato @sym3tri Could one of you approve once again? Had to dismiss the last approval.

**Comment by mxinden** [comment]
Added link to `MAINTAINERS` file.

**Pull request merged**